provd/config.py

```diff
@@ -194,6 +194,9 @@ class ProvdConfigDict(TypedDict):
         'updater': 'default',
         'http_port': DEFAULT_HTTP_PORT,
         'tftp_port': 69,
+        'http_proxied_listen_interface': '127.0.0.1',
+        'http_proxied_listen_port': 18667,
+        'http_proxied_trusted_proxies_count': 1,
```

Comment:
Is it still useful to keep the `num_http_proxies` configuration parameter (on line 203) with this option?

Reply:
Could still be useful for some use cases where a reverse-proxy is needed (like an edge proxy). I intend to remove it in the next PR for HTTP reverse-proxy, when removing the HTTP service from wazo-provd altogether.